docs/Developer-Guide_Build-Preparation.md

```diff
@@ -18,7 +18,36 @@ cd build
 ```
 !!! note
     - Make sure that full path to the build script **does not contain spaces**
-    - For stable branch use `--branch=v24.11`
+    - For stable branch use last point release `--branch=v24.11`
+
+
+    ``` mermaid
```

Comment:
???

Reply:
https://squidfunk.github.io/mkdocs-material/reference/diagrams/?h=mermaid#using-flowcharts

Reply:
ok. It isnt rendered in the preview

Reply:
![image](https://github.com/user-attachments/assets/ae46926c-1c4c-4c4d-a1cf-efe1c755e7f5)